src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -45,14 +46,17 @@ import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
             </ng-container>
         </ul>
         <!--tab content-->
-        <ng-content></ng-content>
+        <ng-container [ngTemplateOutlet]="tabContent.first?.templateRef" ]></ng-container>
```

Comment:
I am projecting only a single content instance, opposed to many clr-tab elements, one of which holds a tab-content.

Reply:
Here is the thing: as much as we tell people to use `*clrIfActive`, we know of at least a handful of internal cases at VMware that don't, and rely on show/hide. With this solution, we can never display the content of anything except the first tab.

Reply:
Acknowledged. no-clrIfActive case implemented and added tests to cover it.